app/public/cantusdata/admin/admin.py

```diff
@@ -53,10 +54,11 @@ class ManuscriptAdmin(admin.ModelAdmin):
 
     @admin.action(description = "Imports the chants associated \
         with the selected manuscript(s)")
+
     def load_chants(self, request, queryset):
-        manuscript_ids = [str(manuscript.pk) for manuscript in queryset]
-        manuscript_ids_str = ','.join(manuscript_ids)
-        return HttpResponseRedirect(f'/admin/cantusdata/manuscript/load_chants/?manuscript_ids={manuscript_ids_str}')
+        manuscript_ids = [manuscript.pk for manuscript in queryset]
```

Comment:
Missing `black` formatting?

Reply:
I thought that github action that I wrote was supposed to take care of this but it looks like it didn't....I will check on that.

Reply:
Oh, the gh-actions will `fail` the build if it's not formatted. 

An automated formatting would involve running `black` on gh-actions and then *committing* the changes. It can be done, but I don't think it is a good idea. Commits that modify the source code should always come from a human.

Reply:
So far, what we have is a `black` "checker" :)

Reply:
makes sense. it passed that check though, so I still need to take a look since it should have failed.

Reply:
can be a separate PR